test/gas.js

```diff
@@ -272,5 +272,92 @@ describe("Gas", function() {
       let gasPrice = '0x2000'
       return testGasExpenseIsCorrect(gasPrice, false, new Web3(Ganache.provider({ mnemonic, gasPrice })))
     })
-  })
+    it('should calculate cumalativeGas and gasUsed correctly when multiple transactions are in a block', function (done){
+      let tempWeb3 = new Web3(Ganache.provider({
+        blockTime: .5, // seconds
+        mnemonic: "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat"
+      }));
+
+      const accounts = [ // configured with mnemonic
```

Comment:
I think these accounts are already available in the `accounts` variable defined previously in the `describe` scope above. Any reason you don't want to just use the ones from that array?